src/components/Timer/TimerReducer.js

```diff
@@ -11,6 +12,8 @@ const timerReducer = (state = initialState, action) => {
             return {...state, playCardDuration: action.playCardDuration };
         case types.SET_VOTE_CARD_TIMER:
             return {...state, voteCardDuration: action.voteCardDuration };
+        case types.SET_STORYTELLER_TIMER:
+            return {...state, storytellerDuration: action.storytellerDuration };
```

Comment:
Look at the inconsistent spacing! (and above) 😄

Reply:
Inconsistent spacing?